src/dbsync/src/sqlite/sqlite_dbengine.cpp

```diff
@@ -25,6 +25,21 @@ SQLiteDBEngine::SQLiteDBEngine(const std::shared_ptr<ISQLiteFactory>& sqliteFact
 SQLiteDBEngine::~SQLiteDBEngine()
 {}
 
+void SQLiteDBEngine::setMaxRows(const std::string& table,
+                                const unsigned long long maxRows)
+{
+    const std::string sql
+    {
+        maxRows ?
+        "CREATE TRIGGER " + table + "_row_count BEFORE INSERT ON " + table +
+        " WHEN (SELECT COUNT(*) FROM " + table + ") >= " + std::to_string(maxRows) +
+        " BEGIN SELECT RAISE(FAIL, 'Too many Rows'); END;"
+        : "DROP TRIGGER " + table + "_row_count"
```

Comment:
Maybe you could just create a new const string variable (rowCountTrigger?) and use it here and in line 34 to avoid repetition and concat times.